pages/user/[id].module.less

```
@@ -0,0 +1,52 @@
+  &.active {
+    background: #2563eb;
+    color: white;
+  }
+}
+
+.custom-tabs {
+  gap: 0.5rem;
+  margin-bottom: 1.5rem;
+  border: none;
+
+  :global(.nav-link) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)